generators/generator-bot-enterprise-assistant/README.md

```diff
@@ -13,29 +61,37 @@ npm install -g @microsoft/generator-bot-enterprise-assistant
 Then generate your new project:
 
 ```bash
-yo @microsoft/bot-assistant-core '{BOT_NAME}'
+yo @microsoft/bot-conversational-core '{BOT_NAME}' -platform '{dotnet|js}' -integration '{functions|webapp}'
 ```
 
-## Packages
-Adaptive bots can utilize the [Azure Bot Framework component model](https://aka.ms/ComponentTemplateDocumentation) to extend their base functionality. The following component packages are included:
+## License
 
-- N/A
+MIT License
 
-## Languages
-English.
+Copyright (c) Microsoft Corporation.
 
-## Azure Resource Deployment
-This template does not rely on any additional Azure Resources.
-
-## Getting To Know Yeoman
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
 
- * Yeoman has a heart of gold.
- * Yeoman is a person with feelings and opinions, but is very easy to work with.
- * Yeoman can be too opinionated at times but is easily convinced not to be.
- * Feel free to [learn more about Yeoman](http://yeoman.io/).
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
 
-## License
-Copyright (c) Microsoft Corporation. All rights reserved.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE
 
-[npm-image]: https://badge.fury.io/js/%40microsoft%2Fgenerator-bot-enterprise-assistant.svg
-[npm-url]: https://www.npmjs.com/package/@microsoft/generator-bot-enterprise-assistant
+[npm-image]: https://badge.fury.io/js/%40microsoft%2Fgenerator-bot-adaptive.svg
```

Comment:
npm-image and npm-url values are incorrect. Please review and double-check for each package.

Reply:
fixed (for each readme)